cli/Sources/TuistOIDC/CIOIDCAuthenticator.swift

```diff
@@ -59,4 +73,34 @@ public struct CIOIDCAuthenticator: CIOIDCAuthenticating {
             audience: "tuist"
         )
     }
+
+    // MARK: - CircleCI
+
+    private var isCircleCIEnvironment: Bool {
+        Environment.current.variables["CIRCLECI"] == "true"
```

Comment:
I'd check the truthiness of the value instead of matching it against the literal `true`. I believe we have a utility for that.